quiz.js

```diff
@@ -0,0 +1,28 @@
+var userHeight = document.getElementById("userHeight");
+var userLeaf = document.getElementById("userLeaf");
+var button = document.getElementById("treeGrown");
+button.addEventListener("click", growTree);
```

Comment:
As per the instructions hen you call this function it should be passed an object with the tree specs in it.